plugins/lookup/passwordstore.py

```diff
@@ -320,7 +323,8 @@ def parse_params(self, term):
 
             if self.backend == 'gopass':
                 self.env['GOPASS_NO_REMINDER'] = "YES"
-            elif os.path.isdir(self.paramvals['directory']):
+
+            if os.path.isdir(self.paramvals['directory']):
                 # Set PASSWORD_STORE_DIR
                 self.env['PASSWORD_STORE_DIR'] = self.paramvals['directory']
             elif self.is_real_pass():
```

Comment:
I'm not sure, but this `elif` probably depends on `if self.backend != 'gopass`?

Reply:
Yes, this is intentional.